### ISSUE TITLE
make fails on macOS Big Sur

### ISSUE THREAD

**Issue by andynameistaken** [FelixKratz/SketchyBar#262]
```
==> make
Last 15 lines from /Users/andy/Library/Logs/Homebrew/sketchybar/01.make:
                                            kAudioObjectPropertyElementName
/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/System/Library/Frameworks/CoreAudio.framework/Headers/AudioHardwareBase.h:331:5: note: 'kAudioObjectPropertyElementName' declared here
    kAudioObjectPropertyElementName         = 'lchn',
    ^
src/volume.c:24:45: error: use of undeclared identifier 'kAudioObjectPropertyElementMain'; did you mean 'kAudioObjectPropertyElementName'?
                                            kAudioObjectPropertyElementMain  };
                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                            kAudioObjectPropertyElementName
/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/System/Library/Frameworks/CoreAudio.framework/Headers/AudioHardwareBase.h:331:5: note: 'kAudioObjectPropertyElementName' declared here
    kAudioObjectPropertyElementName         = 'lchn',
    ^
3 errors generated.
make[1]: *** [bin/volume.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [universal] Error 2
```

- OS: macOS Big Sur 11.7

**Comment by FelixKratz** [comment]
Can you try with latest master?

**Comment by andynameistaken** [comment]
yeah, `brew install sketchybar --HEAD`. Yeah, on Monterey `make` works OOB, I downgraded, because of memory leaks

**Comment by FelixKratz** [comment]
Should be fixed in v2.10.1.
